app/assets/css/tailwind.css

```
@@ -29,6 +29,10 @@ html.no-smooth-scroll {
 }
 /* Tailwind v4 + Nuxt UI v4 theme tokens */
 @theme static {
+  --font-ui:
+    ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)